client/src/components/ChatRoom.svelte

```diff
@@ -84,28 +104,50 @@
 </script>
 
 <div class="chatroom">
-  {#if countdown !== undefined}
+  {#if isStalker}
+    {#if countdown !== undefined}
+      <p class="countdown">{countdown}</p>
+    {/if}
```

Comment:
You can render this once since it doesn't matter if it is the stalker view or chatroom view.